metadata-service/war/src/main/resources/boot/data_platforms.json

```diff
@@ -82,7 +82,7 @@
     "aspect": {
       "datasetNameDelimiter": ".",
       "name": "hana",
-      "displayName": "HANA",
+      "displayName": "SAP HANA",
```

Comment:
Someone recently changed this from SAP HANA to HANA, sighting that this was sufficient to identify the platform. Do you mind unchanging this? Unless you have a strong opinion :) and we can discuss

Reply:
(I see you mentioned this)